### PR TITLE
Add ACES 1.1/1.2 documentation and note where to find configs

### PR DESCRIPTION
**Pull request by omenos**
This PR adds where to find the ACES 1.1 and 1.2 configs to the README as well as the configuration/index page of the site. It also includes the documentation Thomas wrote in his repo, converted to reStructuredText for the main docs.

### CODE REVIEW

**Comment by michdolan** [comment]
FYI: @KelSolaar

**Comment by omenos** [comment]
Not sure if its worth adding more commits in to try and solve the Travis issue, considering the project has switched to Azure Pipelines. It seems to be failing on the docs due to missing components (such as `markupsafe` for Sphinx generation.

**Comment by michdolan** [comment]
Might be a Travis fluke, let's close and reopen to trigger a new build.

File: README.md [hunk 1]
```diff
@@ -54,6 +54,12 @@ The following reference implementations are provided:
   * aces_1.0.2
   * aces_1.0.3
 
+The [Colour Science](https://github.com/colour-science/OpenColorIO-Configs) repository maintained by Thomas Mansencal contains the following additional configurations:
```
Comment (michdolan) [inline README.md:57]: `[Colour Science]` -> `[Colour Science OpenColorIO-Configs]`

`repository` -> `fork`

File: docs/configurations/index.rst [hunk 2]
```diff
@@ -8,6 +8,15 @@ and how to create new ones.
 
 OCIO Configurations can be downloaded here: `.zip <http://github.com/imageworks/OpenColorIO-Configs/zipball/master>`__ `.tar.gz <http://github.com/imageworks/OpenColorIO-Configs/tarball/master>`__ (OCIO v1.0+)
 
+OCIO Configurations for ACES 1.1 and 1.2 can be downloaded from the `Colour Science <https://github.com/colour-science/OpenColorIO-Configs>`_ repository maintained by Thomas Mansencal here:
```
Comment (michdolan) [inline docs/configurations/index.rst:11]: `Colour Science` -> `Colour Science OpenColorIO-Configs`

`repository` -> `fork`

Comment (michdolan) [reply docs/configurations/index.rst:11]: Just for clairty

File: README.md (hunk 1 above)
Comment (michdolan) [reply README.md:57]: Just for clarity

**Comment by michdolan** [comment]
As for the Travis CI failure. It's a bigger issue, possibly related to their VMs drifting since we last used them. It might be worth looking into (though it is unrelated to your PR specifically). As that branch will reach EOL soon though, we'll have to weigh if the effort is worth it.

File: docs/configurations/index.rst (hunk 2 above)
Comment (KelSolaar) [reply docs/configurations/index.rst:11]: and `maintained by Thomas Mansencal here:` --> `maintained by Thomas Mansencal and Michael Parsons here:`

Technically @mlfp is also an official maintainer!

File: README.md (hunk 1 above)
Comment (omenos) [reply README.md:57]: Changes made, just need to push.

File: docs/configurations/index.rst (hunk 2 above)
Comment (omenos) [reply docs/configurations/index.rst:11]: Changes made, just need to push. @KelSolaar I also put that into the README :+1:

**Comment by omenos** [comment]
> Thanks for putting this together! Some minor naming suggestions. Also, it might be worth linking to the Color Science OpenColorIO-Configs fork on this page too:
> https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/RB-1.1/docs/downloads.rst
> What do you think?

I'm open to that, makes plenty of sense to me. How would you like that organized? Between Sample OCIO and Reference Images with a link to the Releases page?

https://github.com/colour-science/OpenColorIO-Configs/releases

**Comment by michdolan** [comment]
Hi @omenos . Now that OCIO v2 is out I would suggest closing this and opening a new PR against the `master` branch. That will avoid the CI issues and allow this to make it into the new OCIO website. Thanks again for the help.

**Comment by omenos** [comment]
Sounds good, @michdolan!

**Pull request closed without merging**